notebooks/faq.clj

```diff
@@ -0,0 +1,62 @@
+
+In contrast, what if
+> Programs are meant to be read by humans and only incidentally for computers to execute. [*D. Knuth*]?
+
+This is the heart of *literate programming*.
+
+The immediate question that flows from this however, is *how*? In TeX systems, the answer was to include executable code within running text. In Mathematica, which largely popularized the *notebook* model, a fully integrated IDE product was built around the underlying language. Jupyter carried on this idea, but universalized the concept by bringing the interaction into the browser.
+
+And yet, for programmers, both of these systems are the wrong way around. The core content and information is in the namespaces and the right level of abstraction is not at arbitrarily chosen *cells*, but rather evaluation at the level of *expressions*. Such is the case, a LisP-style evaluation model is one of the biggest advantages of Woframite, particularly when paired with *notebooks as a namespace*-style literate programming, *e.g.* [clay](https://github.com/scicloj/clay) and [clerk](https://github.com/nextjournal/clerk). When it comes to analysis, being able to interrogate data interactively is often as, if not more, important than being able to run long simulations. With LisP-style literate programming this becomes even more powerful and the other thing to bear in mind is that with literate programming, the docs double as tests!
+
+For more information about the downsides of the *notebook*-model, have a look at this [paper](https://www.microsoft.com/en-us/research/uploads/prod/2020/03/chi20c-sub8173-cam-i16.pdf), as cited by the Clerk document system.
+
+
+**Why not Emmy?**
+
+You *should* use [Emmy](https://github.com/mentat-collective/emmy)! Emmy is a fully Clojure(script) symbolic algebra engine that is built on the [SCMUTILS](https://groups.csail.mit.edu/mac/users/gjs/6946/refman.txt) system, orginally developed by G.J. Sussman and J. Wisdom. Emmy is a very exciting prospect for symbolic manipulation, but is only a subset of what Wolfram is capable of. In the future, we hope to integrate the two more closely. Our current advice though, is that where you have to do a lot of `Solve`-ing (*i.e.* rearranging of equations), or where you need a more powerful simplifier, use Wolframite.
+
+
+**How do I translate Mathematica's syntax sugar?**
+
+Since *Mathematica* hides the internals of the *Wolfram Language* by default, it can sometimes be difficult to work out what functions are being called. If you're in Mathematica, you can use `FullForm@Unevaluated[...]` to get a more understandable (if less concise) expression. Of course, you can also use Wolframite, *i.e.* `(wl/-> ...)` !
+")
```

Comment:
WDYM by `wl/->` ? `wl/->clj`, I assume? Or both `->clj` and `->wl`? I would prefer to be explicit and name them.

Reply:
Good point, probably a typo.